src/main/java/core/basesyntax/Application.java

```diff
@@ -2,6 +2,9 @@
 
 public class Application {
     public static void main(String[] args) {
-        // create three balls using class Lottery and print information about them in console
+        for (int i = 0; i < 3; i++) {
+            Ball ball = Lottery.getRandomBall();
+            System.out.println(ball);
```

Comment:
There is no need to introduce variable for single usage when it's clear what method provides us
```suggestion
            System.out.println(lottery.getRandomBall());
```